.gitmodules

```diff
@@ -13,9 +13,6 @@
 [submodule "tools/firrtl"]
 	path = tools/firrtl
 	url = https://github.com/freechipsproject/firrtl
-[submodule "riscv-tools"]
-	path = toolchains/riscv-tools
-	url = https://github.com/riscv/riscv-tools.git
 [submodule "esp-tools"]
```

Comment:
nit: if `riscv-tools` is now labeled as `toolchains/riscv-tools` can we switch `esp-tools` to match the same `folder/name` structure (and other similar submodules that are under a folder)

Reply:
done